### PR TITLE
Improved some text entries

### PR DESCRIPTION
**Pull request by siddhpant**

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-themes/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/Translations.xml [hunk 1]
```diff
@@ -107,7 +107,7 @@ platform.flamingo.themes.category.panels=Panels
 platform.flamingo.themes.category.breadcrumb=Breadcrumb
 platform.flamingo.themes.category.advanced=Advanced
 platform.flamingo.themes.defaultvalue=Default
-platform.flamingo.themes.lessCode.description=In this field, you can directly write LESS code that will be added into the skin.
+platform.flamingo.themes.lessCode.description=In this field, you can directly write new LESS code for the skin.
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-themes/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/Translations.xml]: It was better before. Please revert this change.

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Translations.xml [hunk 2]
```diff
@@ -41,10 +41,10 @@
   <hidden>true</hidden>
   <content>help.tipsPanel.title=Tips
 help.tipsPanel.noTipMessage=No Tip available!
-help.tipsPanel.gettingStartedTip=If you're starting with XWiki, check out the [[Getting Started Guide&gt;&gt;http://enterprise.xwiki.org/xwiki/bin/view/GettingStarted/]].
+help.tipsPanel.gettingStartedTip=If you're a new XWiki user, check out the [[Getting Started Guide&gt;&gt;http://enterprise.xwiki.org/xwiki/bin/view/GettingStarted/]].
 help.tipsPanel.jumpToPageTip=Jump to any page in the wiki with Ctrl+G or Meta+G.\nSee [[more shortcuts&gt;&gt;http://platform.xwiki.org/xwiki/bin/view/Features/KeyboardShortcuts]].
 help.tipsPanel.improveXWikiTip=Did you know that you can improve XWiki? Take 5 minutes to [[fill this survey&gt;&gt;http://www.xwiki.org/xwiki/bin/view/Survey/ProductFeedback]] and help this open source project!
-help.tipsPanel.breadcrumbsTip=You can click on the arrows next to the breadcrumb elements to quickly navigate to sibling and children pages.
+help.tipsPanel.breadcrumbsTip=You can click on the arrows next to the breadcrumb elements to quickly navigate sibling and children pages.
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Translations.xml]: The user is not navigating the pages (i.e. a ship, a boat, etc.), but navigating to the pages (i.e. to a location, destination). Please revert it.

File: xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemesCode/Translations.xml [hunk 3]
```diff
@@ -45,9 +45,9 @@ platform.icon.picker.loading=Loading
 rendering.macro.iconPicker.name=Icon Picker
 rendering.macro.iconPicker.description=Select an icon within the XWiki icon set.
 rendering.macro.iconPicker.parameter.id.name=Id
-rendering.macro.iconPicker.parameter.id.description=DOM id of the input field where the picker will apply
+rendering.macro.iconPicker.parameter.id.description=DOM id of the input field where the icon would be set
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemesCode/Translations.xml]: That's not really what it does. It's about the input where the icon picker widget will attach to, not about the location of an icon. Please revert.

File: xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemesCode/Translations.xml [hunk 4]
```diff
@@ -45,9 +45,9 @@ platform.icon.picker.loading=Loading
...
 rendering.macro.iconPicker.parameter.class.name=Class
-rendering.macro.iconPicker.parameter.class.description=CSS class of inputs where the picker will apply
+rendering.macro.iconPicker.parameter.class.description=CSS class of inputs where the icon would be set
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-icon/xwiki-platform-icon-ui/src/main/resources/IconThemesCode/Translations.xml]: Same as above.

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml [hunk 5]
```diff
@@ -42,12 +42,12 @@
...
-Invitation.WebHome_from_address.hint=The email address used to send invitation emails from.
-Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server, only when authentication is required.
-Invitation.WebHome_smtp_server_username.hint=The user name to connect to the SMTP server, only when authentication is required.
+Invitation.WebHome_from_address.hint=The email address used to send the invitation emails.
+Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server.
+Invitation.WebHome_smtp_server_username.hint=The username to connect to the SMTP server.
 Invitation.WebHome_smtp_port.hint=The port of the SMTP server to use.
 Invitation.WebHome_smtp_server.hint=The SMTP server to send to.
-Invitation.WebHome_javamail_extra_props.hint=The list of additional Java Mail properties to use.</content>
+Invitation.WebHome_javamail_extra_props.hint=List of additional JavaMail properties to use.</content>
```
Comment (Enygma2002) [inline xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: Please keep the `The` start, to be consistent with the previous hints.

Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: "The" before list seems wrong. For eg. the title of this (and all other) Wikipedia page starts directly from list : https://en.m.wikipedia.org/wiki/List_of_Prime_Ministers_of_India

File: xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-themes/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/Translations.xml (hunk 1 above)
Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-themes/xwiki-platform-flamingo-theme-ui/src/main/resources/FlamingoThemesCode/Translations.xml]: Okay

File: xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Translations.xml (hunk 2 above)
Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/Help/Translations.xml]: Okay

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml (hunk 5 above)
Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: Done.

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml [hunk 6]
```diff
@@ -42,12 +42,12 @@
   <content>invitation.applicationsPanelEntryLabel=Invitation
 
 # Invitation Administration Section
-Invitation.WebHome_from_address.hint=The email address used to send invitation emails from.
-Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server, only when authentication is required.
-Invitation.WebHome_smtp_server_username.hint=The user name to connect to the SMTP server, only when authentication is required.
+Invitation.WebHome_from_address.hint=The email address used to send the invitation emails.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: I think this loose a bit the meaning, we don't know anymore if it's the target mail or the from mail.

Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: 'send'

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml [hunk 7]
```diff
@@ -42,12 +42,12 @@
...
+Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: I don't really see this as an improvement, an important information has been removed.

Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: It will ask for it only when authenticating.

Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: That's implied. Password is for authentication.

File: xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/PanelsCode/Translations.xml [hunk 8]
```diff
@@ -39,12 +39,12 @@
...
   <syntaxId>plain/1.0</syntaxId>
   <hidden>true</hidden>
-  <content>platform.panels.applications.helper=Here, you can select which applications are available in the Applications Bar (on the left, by default). Any new application will be displayed there. If you want to remove some of them, you just need to drag and drop the icon from the first box to the other. You can also re-order the applications as you want. Don't forget to save your changes!
+  <content>platform.panels.applications.helper=Here, you can select which applications are displayed in the Applications Bar (on the left, by default). Any new application will be displayed there. If you want to remove some of them, you just need to drag and drop the icon from the first box to the other. You can also re-order the applications as you want. Don't forget to save your changes!
 platform.panels.applications.displayedapps=Applications in the bar
 platform.panels.applications.blacklistedapps=Applications not in the bar
 platform.panels.applications.save=Save
 platform.panels.applications.revert=Reset
-platform.panels.applications.ajax.success=The configuration has been saved.
+platform.panels.applications.ajax.success=Saved!
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/PanelsCode/Translations.xml]: I don't see anything wrong in the previous message and this is a big change.

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml (hunk 6 above)
Comment (siddhpant) [inline xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: done

Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: done

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml (hunk 7 above)
Comment (tmortagne) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: I don't agree, this is not the goal of the task. You are supposed to only work on the English, not change the meaning of a message.

File: xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/PanelsCode/Translations.xml (hunk 8 above)
Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/PanelsCode/Translations.xml]: Verbose?

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml [hunk 9]
```diff
@@ -42,12 +42,12 @@
   <content>invitation.applicationsPanelEntryLabel=Invitation
 
 # Invitation Administration Section
-Invitation.WebHome_from_address.hint=The email address used to send invitation emails from.
-Invitation.WebHome_smtp_server_password.hint=The password to connect to the SMTP server, only when authentication is required.
```
Comment (tmortagne) [inline xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: Please put it back.

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml (hunk 7 above)
Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: ..okay

File: xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml (hunk 9 above)
Comment (siddhpant) [reply xwiki-platform-core/xwiki-platform-invitation/xwiki-platform-invitation-ui/src/main/resources/Invitation/Translations.xml]: Done.

**Comment by tmortagne** [comment]
Thanks for your contribution !
